README.md

```diff
@@ -64,6 +64,26 @@
 # Contributing
 ❤️Legesher relies on the passionate members of its community (both developer and non-developer alike) to keep delivering impactful tools to people all over the world.
 
+**🌟 Add your name to our website!**  
+Before you open the pull request for your changes, add your name to our website!
+
+**1️⃣ Navigate to and open index.html**
+
+**2️⃣ Scroll down to the contributors section** 
+```
+<section class="contributors section">
+```
+
+**3️⃣ Did you do technical work or non-technical?**
+If was technical put your name under the column with the chip svg icon!
+```
+<div class="contributors-column technical">
+```
+If was non-technical put your name under the column with the lightbulb svg icon!
+```
+<div class="contributors-column non-technical">
+```
```

Comment:
This is a great way of helping contributors understand what to do! Could you add in a little more detail, as some of our contributors don't know html. 😄 Thinking of something along these lines:

``` html 
<div class="contributors-column technical">
  ...
  <a href="https://github.com/YOURUSERNAME">YOUR NAME</a> 
  ...
</div>
```

Reply:
Did we want to take this out of `README.md` and create `contributing.md`?